### PR TITLE
Added OnDiskBitmap functionality

### LINKED ISSUE #8: Add OnDiskBitmap support
This one should be pretty easy. Open a bitmap with Pillow, and return in Displayio Bitmap format.

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #8.

### CODE REVIEW

File: displayio/colorconverter.py [hunk 1]
```diff
@@ -122,3 +135,16 @@ def dither(self, value):
         if not isinstance(value, bool):
             raise ValueError("Value should be boolean")
         self._dither = value
+
+    @property
+    def rgba(self):
```
Comment (tannewt) [inline displayio/colorconverter.py:140]: Why are you adding this? Doing so means people may not be able to move their code back to CircuitPython.

Comment (makermelissa) [reply displayio/colorconverter.py:140]: This was to specify that it should return the value in rgba format and is really for internal use. The other option is that I could just directly set the private variable from the Tilegrid class, which is fine with me.

Comment (makermelissa) [reply displayio/colorconverter.py]: Ok, I removed the property and am just accessing the variable directly so users shouldn't make incompatible code now.

**Comment by makermelissa** [comment]
This not only supports bitmaps, but also gif, png (including transparent), and jpeg since Pillow is handling all of this.

**Pull request merged**